dvc/repo/experiments/queue/base.py

```diff
@@ -21,7 +21,7 @@
 from funcy import retry
 
 from dvc.dependency import ParamsDependency
-from dvc.env import DVC_EXP_BASELINE_REV, DVC_EXP_NAME
+from dvc.env import DVC_EXP_BASELINE_REV, DVC_EXP_NAME, DVC_ROOT
```

Comment:
I am confused by this already existing.
Either I am missing something or this is dead code:

https://github.com/iterative/dvc/blob/ef260eaf2e2c207edc26505f510e4f79b11ba6a4/dvc/stage/__init__.py#L288-L295

Reply:
Looks like dead code from checkpoints: https://github.com/iterative/dvc/pull/5257

Reply:
I'll try and remove it in a separate PR.